telegram/utils/request.py

```diff
@@ -136,13 +136,19 @@ def _parse(json_data):
         raise TelegramError('Invalid server response')
 
     if not data.get('ok'):
-        if data.get('parameters') and data['parameters'].get('migrate_to_chat_id'):
-            raise ChatMigrated(str(data['parameters']['migrate_to_chat_id']))
-        if data.get('description'):
-            return data['description']
+        description = data.get('description')
+        parameters = data.get('parameters')
+        if description:
+            if parameters:
+                migrate_to_chat_id = parameters.get('migrate_to_chat_id')
+                if migrate_to_chat_id:
+                    raise ChatMigrated(description, chat_id=int(migrate_to_chat_id))
+            else:
+                return description
```

Comment:
The above code is correct with the JSON reply.
However, it is better to be more liberal with the way we interpret the replies from Telegram servers, especially that they like making changes without any prior notification.
So... please do something more similar to your original commit:
```
if parameters:
    if migrate... :
        raise
if description:
    return description
```